application/models/Plugin.php

```diff
@@ -87,8 +87,12 @@ public function setLoadError(array $error)
      */
     public function isCompatible()
     {
-        $config = $this->getExtensionConfig();
-        return $config->isCompatible();
+        try {
+            $config = $this->getExtensionConfig();
+            return $config->isCompatible();
+        } catch (\Exception $ex) {
```

Comment:
Which exception are you catching here?

Reply:
Any exception from the getExtensionConfig or the isCompatible method.
Example: 'Missing configuration file for plugin %s, looked in location %s',

Reply:
Hm, do we want to silence that? Try-catch usually makes better sense higher up in the stacktrace, e.g. in a controller or such.

Reply:
By returning `false` we lose important information.

Reply:
There could be lot of malformations in config.xml files, missing config.xml files and even other situations not thought.
As so, any of those situations can be considered as a non compatibility.
By that is that we return false.

If not returning false, we should look up every usage of ->isCompatible() and add try/catch.
Still, not sure if that hadling would be much different.

Maybe, to inform the error we could add an optional parameter with the exception caught?